npm-shrinkwrap.json

```diff
@@ -366,78 +380,78 @@
     },
     "nodemailer": {
       "version": "1.3.0",
-      "from": "nodemailer@^1.1.1",
+      "from": "https://registry.npmjs.org/nodemailer/-/nodemailer-1.3.0.tgz",
```

Comment:
After a second look at this, `version` and `resolved` haven't changed, so it seems to be ok. NTC.